src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -262,6 +262,9 @@ TAP_ESC::init()
 		return ret;
 	}
 
+#ifdef CONFIG_ARCH_BOARD_AEROFC_V1
```

Comment:
This needs to be revisited when final HW is out there. Right now it just returns 0 when trying to read back the values.